tests/EmailReplyParser/Tests/Parser/EmailParserTest.php

```diff
@@ -264,6 +264,12 @@ public function testCustomQuoteHeader2()
         $this->assertTrue($fragments[1]->isQuoted());
     }
 
+    public function testCustomSigRegex()
+    {
+        $regex = $this->parser->getSignatureRegex();
+        $this->parser->setSignatureRegex($regex);
```

Comment:
seems not other than the method call doesn't fail. what you suggest to test there?